person_class.rb

```diff
@@ -26,5 +28,9 @@ def correct_name
     @name
   end
 
+  def add_rentals(date, book)
+    Rental.new(date, self, book)
+  end
+
   private :of_age?
 end
```

Comment:
- [ ] `README.md` file: To make your documentation looks more professional, kindly make sure you fix your broken image in your README and try to remove all placeholder text on each section(if exists). Below you can fin some example of them, you can go through your entire file and remove all of them.
### Broken image
![Screenshot from 2023-04-12 12-54-51](https://user-images.githubusercontent.com/83042742/231529659-3668ce27-0e6c-4443-ada1-3fefd4dc825f.png)

### Example of Images with placeholder textx to remove
| image | image | image |
| ------------- | --------------- | ------------- |
| ![Screenshot from 2023-04-12 12-57-06](https://user-images.githubusercontent.com/83042742/231530332-7d078a7b-5e9b-4789-a37e-2be234f3bd07.png) | ![Screenshot from 2023-04-12 12-56-39](https://user-images.githubusercontent.com/83042742/231530369-1531a99c-6f1c-4cd1-89a5-774b29059bf0.png) | ![Screenshot from 2023-04-12 12-55-34](https://user-images.githubusercontent.com/83042742/231530418-5191ffa1-7ea0-4813-b3f8-a4ff0efa0fc5.png) |